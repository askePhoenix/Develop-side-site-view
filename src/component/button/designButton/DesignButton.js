import React from 'react'

function DesignButton() {
    return (
        <input type="button">
            default
        </input>
    );
}

export default DesignButton;