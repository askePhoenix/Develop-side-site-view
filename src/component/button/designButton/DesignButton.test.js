import React from 'react';
import renderer from 'react-test-renderer';
import DesignButton from './DesignButton';

test('Link changes the class when hovered', () => {
    const component = renderer.create(
        <DesignButton />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
