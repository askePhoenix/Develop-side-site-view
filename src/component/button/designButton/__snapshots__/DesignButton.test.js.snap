// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<input
  type="button"
>
  default
</input>
`;
